scraper_classes.py

```diff
@@ -0,0 +1,119 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+class Courses:
+    # basic class info
+    __class_code = None
+    __year = None
+    __semester = None
+    class_id_list = []
+    min_units_list = []
+    max_units_list = []
+    description_list = []
+    URL = None
+
+    def __init__(self, url_):
+        self.URL = url_
+
+    def parse(self):
+        global __class_code
+        global __year
+        global __semester
+        global URL
```

Comment:
why is `global` needed?

Reply:
ah oops that's leftover from when I didn't have the function defined as part of a Courses class